site/docs/get-started/index.md

```diff
@@ -9,16 +9,16 @@ virtual machines, public clouds and containers.
 
 With ManageIQ you will be able to:
 
-* Continuously discover the latest state of your environment.
+* **Continuously discover** the latest state of your environment.
 * Implement **self service** for your end users.
 * Enforce **compliance** across the environment.
-* Optimize the performance and utilization of you environment.
+* **Optimize** the performance and utilization of you environment.
 
 ## Following this Guide
 
 In order to follow this guide, you will need:
 
-* Somewhere to run the ManageIQ software.
+* Some place to run the ManageIQ software.
```

Comment:
Should we be clear and say "a _system_ to run the ManageIQ software" and  "an infrastructure to manage"?